lib/core/styles/theme_helper.dart

```diff
@@ -17,6 +17,7 @@ class ThemeHelper {
     visualDensity: VisualDensity.adaptivePlatformDensity,
     accentColor: lightColor,
     primaryColor: darkColor,
+    canvasColor: ThemeHelper.darkColor,
```

Comment:
Why do we need this color if it is the same as the darkColor?

Reply:
We need to canvasColor for changing drawer background color.

![image](https://user-images.githubusercontent.com/21019611/107847105-d43d3f00-6df9-11eb-917a-56c34dfd55e5.png)